sourcegraph/sourcegraph
# Skeleton UI: New Setup Wizard > Header

Closes #46900. Part of #46512.

- Removes outdated `Step` content (new design in works)
- Sets up header for new design
- Experimental feature flag provides route only in developer environment

<img width="1295" alt="image" src="https://user-images.githubusercontent.com/59381432/214711374-98b8999d-e0bd-45db-8b75-66d8aac93441.png">

## Test plan
- Visit setup wizard in developer mode (`/setup`)
- Ensure if `/setup` isn't available outside the developer environment

## App preview:

- [Web](https://sg-web-becca-skeleton-header.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
